scripts/delayed.js

```diff
@@ -1,7 +1,60 @@
 // eslint-disable-next-line import/no-cycle
-import { sampleRUM } from './lib-franklin.js';
+import { sampleRUM, getMetadata } from './lib-franklin.js';
 
 // Core Web Vitals RUM collection
 sampleRUM('cwv');
 
 // add more delayed functionality here
+
+/**
+ * Define the environment based on the hostname
+ * @param {*} hostname
+ * @returns
+ */
+export function pageEnvironment(hostname) {
+  switch (hostname) {
+    case 'www.zeiss.de':
+    case 'www.zeiss.com':
+      return 'prod_publish';
+    case 'localhost':
+      return 'local_publish';
+    default:
+      return 'publish';
+  }
+}
+
+export function pathItem(pathname, index) {
+  // Capture the path item at the given index
+  return pathname.split('/')[index] || '';
+}
+
+function loadGoogleTagManager() {
+  // Initialize the data layer
+  /* eslint no-undef: "error" */
+  const url = new URL(window.location.href);
+  const locale = getMetadata('locale');
+  const pageCountry = {
+    en: 'INT',
+    de: 'DE',
+  };
+  const conf = {
+    pageArea: 'web',
+    pageCountry: pageCountry[locale],
+    pageEnvironment: pageEnvironment(url.hostname),
+    pageIdentifier: 'main',
+    pageLanguage: locale,
+    pageLocation: 'ALL_General-Article_002',
+    pageName: url.pathname,
+    pagePool: pathItem(url.pathname, 4),
```

Comment:
Other option is to set via bulk-metadata just in case the folder structure changes later, we'd not have to update it in code here and the author can directly set it in the metadata spreadsheet.

Reply:
I suggest to add this level of flexibility when we really need it (YAGNI)